ui/app/AppLayouts/Chat/controls/community/PermissionsDropdown.qml

```diff
@@ -15,6 +15,8 @@ StatusDropdown {
     property int mode: PermissionsDropdown.Mode.Add
     property int initialPermissionType: PermissionTypes.Type.None
 
+    property bool disableAdminPermission: false
```

Comment:
Why the double negative? :)

Reply:
My thinking process about that: disabling admin permission is an exception here, this exception is not active by default so it's set to false. When we want to "activate" that exception, we need to set it to true.

Enabling exception by setting flag to false seems slightly less natural for me. For me it's matter of taste/style/whatever :)

On the other hand changing it to `enableAdminPermission: true` results in double positive :upside_down_face: 

Wdyt? Can we keep it as it is or you want it be changed? I'm 50/50.

Reply:
imho it is easier to fall into mistakes with the double negative :face_with_spiral_eyes: 
Moreover I don't see this case as an exception, we can have 2 different user types, admin or owner, any of them is an exception, it is just a case, so, for me `adminPermissionEnabled` is fine and natural.

Reply:
Ok guys, you are right, I will fix it :)